app/src/main/res/xml-eu/ext_kbd_bottom_row_ng.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think it would make much more sense to have those coming from the language pack.